hotness/consumers.py

```diff
@@ -297,8 +351,8 @@ def handle_buildsys_scratch(self, msg):
         text2 = "%s %s" % (subtitle, url)
 
         # Followup on bugs we filed
-        if task_id in self.triggered_task_ids:
-            bugs.append((self.triggered_task_ids.pop(task_id), text1))
+        #if task_id in self.triggered_task_ids:
+        #    bugs.append((self.triggered_task_ids.pop(task_id), text1))
```

Comment:
I think we only used fedpkg to pull down the tarball and any patches.  We would interact with the koji xmlrpc api to kick off the build -- and it's possible (if a little tricky) to use that koji api to pull down all the builds artifacts at the end.  I expect that this is what rebase-helper is already doing.